crypto/boxed_message.go

```diff
@@ -56,49 +61,55 @@ func (b *boxedMessage) Load(from []byte) error {
 		return fmt.Errorf("invalid message format")
 	}
 	matches := allMatches[0]
-	if len(matches) != 5 {
+	if len(matches) != 6 {
 		return fmt.Errorf("invalid message format")
 	}
 
-	ssver = matches[1]
-	spub = matches[2]
-	snonce = matches[3]
-	sbox = matches[4]
+	ssver = matches[2]
```

Comment:
we've added a capture group, so this shifts by one.

probably named capture group would make things clearer?

Reply:
.. would complicate the code, though ;)